tests/app/dao/test_annual_billing_dao.py

```diff
@@ -41,3 +41,16 @@ def test_dao_update_annual_billing_for_future_years(notify_db_session, sample_se
     assert dao_get_free_sms_fragment_limit_for_year(sample_service.id, current_year) is None
     assert dao_get_free_sms_fragment_limit_for_year(sample_service.id, current_year + 1).free_sms_fragment_limit == 9999
     assert dao_get_free_sms_fragment_limit_for_year(sample_service.id, current_year + 2).free_sms_fragment_limit == 9999
+
+
+def test_dao_get_annual_billing(notify_db_session, sample_service):
+    """
+    Get all AnnualBilling instances associated with a given Service.
+    """
+
+    query = select(func.count()).select_from(AnnualBilling).where(AnnualBilling.service_id == sample_service.id)
+    assert notify_db_session.session.scalars(query).first() == 0
```

Comment:
Standardized naming please. 
```suggestion
    stmt = select(func.count()).select_from(AnnualBilling).where(AnnualBilling.service_id == sample_service.id)
    assert notify_db_session.session.scalars(stmt).first() == 0
```